### DISCUSSION TITLE
provide a way to measure remote writes

### DISCUSSION THREAD

**Discussion by meangrape** [prometheus/prometheus#10407] [Ideas]
Hi!

There is a `prometheus_api_remote_read_queries` metric. Would it possible to have a metric that measures the amount of remote writes?

Thanks!

**Comment by roidelapluie** [comment]
I would expect `prometheus_remote_storage_sent_batch_duration_seconds_count` to cover this. WDYT @csmarchbanks @cstyan ?

**Reply by cstyan** [reply]
Yes. There's metrics for number of sends, number of retries, bytes sent, etc.
